apm-sniffer/apm-sdk-plugin/mssql-jdbc-plugin/src/main/resources/skywalking-plugin.def

```diff
@@ -16,3 +16,5 @@
 
 mssql-jdbc=org.apache.skywalking.apm.plugin.mssql.jdbc.define.DriverInstrumentation
 mssql-jdbc=org.apache.skywalking.apm.plugin.mssql.jdbc.define.ConnectionInstrumentation
+mssql-jdbc=org.apache.skywalking.apm.plugin.mssql.jdbc.define.PreparedStatementInstrumentation
+mssql-jdbc=org.apache.skywalking.apm.plugin.mssql.jdbc.define.StatementInstrumentation
```

Comment:
mssql plugin doesn't instrument statement before? I am a little confused.

Reply:
In the previous packaging mode, statement was packaged by `org.apache.skywalking.apm.plugin.jdbc.JDBCStatementInterceptor`.